### ISSUE TITLE
SyntaxWarning with Python 3.8

### ISSUE THREAD

**Issue by swt2c** [pymupdf/PyMuPDF#354]
When building with Python 3.8 on Fedora, I'm seeing this SyntaxWarning:

```
Bytecompiling .py files below /builddir/build/BUILDROOT/python-PyMuPDF-1.16.0-1.fc32.x86_64/usr/lib64/python3.8 using /usr/bin/python3.8
/usr/lib64/python3.8/fitz.py:3166: SyntaxWarning: "is" with a literal. Did you mean "=="?
/usr/lib64/python3.8/fitz.py:3166: SyntaxWarning: "is" with a literal. Did you mean "=="?
```

**Comment by JorjMcKie** [comment]
Weird - the lines around that number have no  " is " statement when I download from PyPI.
But wait - are you generating ``fitz.py`` with an SWIG version other than 4.0.0? Then you get a different source layout of course.
Please show me the environment of that line number please - so I can find the origin in my source ...

**Comment by swt2c** [comment]
My bad, I forgot that's a generated file.  Looks like I'm using swig 4.0.1.  This is the context of the error:
```
    def __repr__(self):
        m = "closed " if self.isClosed else ""
        if self.stream is None:
            if self.name is "":
                return m + "fitz.Document(<new PDF, doc# %i>)" % self._graft_id
            return m + "fitz.Document('%s')" % (self.name,)
        return m + "fitz.Document('%s', <memory, doc# %i>)" % (self.name, self._graft_id)
```

**Comment by swt2c** [comment]
Looks like this is the origin:
https://github.com/pymupdf/PyMuPDF/blob/master/fitz/fitz.i#L2427

**Comment by JorjMcKie** [comment]
Ok, thanks found it and corrected.

**Comment by JorjMcKie** [comment]
should be fixed in v1.16.1 now
